src/items.md

```diff
@@ -470,21 +472,20 @@ let p = Point {x: 10, y: 11};
 let px: i32 = p.x;
 ```
 
-A _tuple struct_ is a nominal [tuple type], also defined with
-the keyword `struct`. For example:
+A _tuple struct_ is a nominal [tuple type], also defined with the keyword
+`struct`. For example:
 
-[struct type]: types.html#struct-types
-[tuple type]: types.html#tuple-types
+[struct type]: types.html#struct-types [tuple type]: types.html#tuple-types
```

Comment:
Same here with does this actually break links? We probably want them on multiple lines even if they don't.